src/kernels/jupyter/finder/remoteKernelFinder.ts

```diff
@@ -50,21 +50,22 @@ import { IApplicationEnvironment } from '../../../platform/common/application/ty
 import { KernelFinder } from '../../kernelFinder';
 import { RemoteKernelSpecsCacheKey, removeOldCachedItems } from '../../common/commonFinder';
 import { IExtensionSingleActivationService } from '../../../platform/activation/types';
+import { ContributedKernelFinderKind } from '../../internalTypes';
 
 // Even after shutting down a kernel, the server API still returns the old information.
 // Re-query after 2 seconds to ensure we don't get stale information.
 const REMOTE_KERNEL_REFRESH_INTERVAL = 2_000;
 
-// This class searches for a kernel that matches the given kernel name.
-// First it searches on a global persistent state, then on the installed python interpreters,
-// and finally on the default locations that jupyter installs kernels on.
+// This class finds kernels from the current selected Jupyter URI
 @injectable()
 export class RemoteKernelFinder implements IRemoteKernelFinder, IExtensionSingleActivationService {
     /**
      * List of ids of kernels that should be hidden from the kernel picker.
      */
     private readonly kernelIdsToHide = new Set<string>();
-    kind: string = 'remote';
+    kind = ContributedKernelFinderKind.Remote;
+    id: string = 'currentremote';
+    displayName: string = 'Current Remote';
```

Comment:
in local kernel finder this is localized, but not here

Reply:
Added the string, but forgot to link it in here!